### DISCUSSION TITLE
Verbindung SAE mit Sunny Home Manager / Sunny Portal

### DISCUSSION THREAD

**Discussion by teuti** [camueller/SmartApplianceEnabler#523] [Installation und Betrieb]
Moin zusammen,

ich habe mir gestern auf einem Raspi SAE mittels Skript installiert. Als das erfolgreich durchgelaufen ist, habe ich als Geräte eine neue Steckdose mit Tasmota eingerichtet. Über einen HTTP-Schalter kann ich die Steckdose ein- und ausschalten; auf der Status-Seite sehe ich die Ampel zu dem Gerät und dort kann ich die Steckdose ebenfalls einschalten. Soweit also erst mal so gut.

Das Problem ist jetzt, dass ich den SAE nicht mit dem Sunny Portal koppeln kann. Wenn ich im Sunny Portal unter "Konfiguration - Geräteübersicht" den Reiter "Übersicht Neugeräte" und dort den Button "Geräte aktualisieren" anklicke, dann passiert genau nichts. Er sagt mir, dass er keine Gerate finden konnte. Bisher ist dort auch noch kein Gerät vorhanden - die Grenze sprenge ich also nicht.

Kann mir einer sagen, wo ich einen Fehler gemacht habe?

Gruß,
teuti

**Comment by camueller** [comment]
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md

**Reply by teuti** [reply]
Die QA habe ich mir natürlich auch schon durchgelesen und ich wollte noch erwähnt haben, dass ich den Home Manager bereits neugestartet habe.

**Reply by camueller** [reply]
Das heisst, dass Du im SAE-Log die Einträge für die Abfragen durch den SHM gefunden hast?

**Reply by teuti** [reply]
Du meinst diese Einträge?

2023-05-05 11:01:39,446 INFO [main] d.a.s.s.w.SempController [SempController.java:61] SEMP controller created.
2023-05-05 11:01:41,717 DEBUG [main] d.a.s.Application [Application.java:103] Starting SEMP discovery ...
2023-05-05 11:01:41,859 INFO [main] d.a.s.s.d.SempDiscovery [SempDiscovery.java:58] SEMP UPnP will redirect to http://192.168.1.6:8080
2023-05-05 11:01:41,872 DEBUG [main] d.a.s.Application [Application.java:106] ... SEMP discovery started

**Reply by camueller** [reply]
Nein. Du hattest doch geschrieben, dass Du die Q&A gelesen hast.

https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp1
